man/sbd.8.pod

```diff
@@ -394,6 +394,30 @@ a fencing message by the cluster.
 
 =back
 
+=head2 query-watchdog
+
+Example usage:
+
+	sbd query-watchdog
+
+Check for available watchdog-devices and print some info.
+
+B<Warning>: This command will arm the watchdog during query and if your
+watchdog refuses disarming (nowayout-parameter set for kernel-module)
+this might reset your system.
+
+=head2 test-watchdog
+
+Example usage:
+
+	sbd test-watchdog [-w /dev/watchdog3]
+
+Test the watchdog-device selected.
```

Comment:
"specified watchdog device"
 
maybe add "(/dev/watchdog by default)" if that's true